ios/FluentUI/Table View/TableViewTokens.swift

```diff
@@ -4,172 +4,329 @@
 //
 import UIKit
 
-open class TableViewCellTokens: ControlTokens {
-    /// The background color of the TableView.
-    open var backgroundColor: DynamicColor {
-        .init(light: globalTokens.neutralColors[.white],
-              dark: globalTokens.neutralColors[.black])
-    }
+public class TableViewCellTokenSet: ControlTokenSet<TableViewCellTokenSet.Tokens> {
```

Comment:
Did you want to also rename this file?

Reply:
Good catch. This didn't happen automatically because the file name didn't match the class. Changing it manually to match for the future.